app/browser/tabs.js

```diff
@@ -1019,9 +1020,10 @@ const api = {
           createProperties.parent_partition = ''
         }
         if (createProperties.isTor) {
+          // TODO(riastradh): Duplicate logic in app/filtering.js.
           createProperties.isolated_storage = true
           createProperties.parent_partition = ''
-          createProperties.tor_proxy = 'socks5://127.0.0.1:9050'
+          createProperties.tor_proxy = getTorSocksProxy()
```

Comment:
[minor] this function gets called every time a new tab is created. you could declare `const TOR_SOCKS_PROXY = getTorSocksProxy()` at the beginning of this file and just use the constant here since it won't change for a given build of Brave